src/securejoin/bob.rs

```diff
@@ -100,6 +100,9 @@ pub(super) async fn handle_auth_required(
                 let chat_id = bobstate.joining_chat_id(context).await?;
                 chat::add_info_msg(context, chat_id, &msg, time()).await?;
             }
+            bobstate
+                .notify_peer_verified(context, message.timestamp_sent)
```

Comment:
`notify_peer_verified()` doesn't look like a good function name. It not only notifies the user, but also sets the chat protection. Maybe just `set_peer_verified()`?